cluster/get-kube.sh

```diff
@@ -37,6 +37,17 @@
 #    * arm
 #    * arm64
 #
+#  Set KUBERNETES_NODE_PLATFORM to choose the platform for which to download
+#  the node binaries. If not set (the default) no node binaries will be
+#  downloaded. The options are:
+#    * linux
+#    * windows
+#
+#  Set KUBERNETES_NODE_ARCH to choose the node architecture to download:
```

Comment:
it looks like this only has an effect if `KUBERNETES_NODE_PLATFORM` is also set?

if I only set `KUBERNETES_NODE_PLATFORM` to `arm` it doesn't seem like this will do anything?

Reply:
Yes, because realistically, unless you want a different node platform, you can simply build and upload `kubernetes-server-linux-<arch>.tar.gz`. You will only want to build a separate node tarball if the platform is not linux